docs/modules/ROOT/pages/reference/extensions/atlasmap.adoc

```diff
@@ -31,3 +31,27 @@ Please refer to the above link for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== allowContextMapAll option in native mode
+
+The `allowContextMapAll` option is not supported in native mode as it requires reflective access to security sensitive camel core classes such as
+`CamelContext` & `Exchange`. This is considered a security risk and thus access to the feature is not provided by default.
+
+== Additional Camel Quarkus configuration
+
+Note that you can perform mapping for these types of data:
+
+* Java
+* XML
+* Json
+* CSV
+
+The _ADM archive file_ or _mapping definition JSON file_ used in the _AtlasMap route_ should be located in the _resources_ repository.
+In native mode, include this file for native build. Example:
+[source,properties]
+----
+quarkus.native.additional-build-args = -H:IncludeResources=atlas-mapping.json
+----
```

Comment:
Maybe one could use https://quarkus.io/guides/building-native-image#quarkus-native-pkg-native-config_quarkus.native.resources.includes instead.

Reply:
thanks, I didn't know this one !!